app/src/main/java/com/framgia/moviedb/data/repository/MoviesContentAsyncTask.java

```diff
@@ -0,0 +1,72 @@
+package com.framgia.moviedb.data.repository;
+
+import android.os.AsyncTask;
+import com.framgia.moviedb.data.model.GroupMovie;
+import com.framgia.moviedb.data.model.Movie;
+import com.framgia.moviedb.data.source.MoviesDataSource;
+import com.framgia.moviedb.utils.Constants;
+import java.util.ArrayList;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class MoviesContentAsyncTask extends AsyncTask<String, Void, List<GroupMovie>> {
+    private MoviesDataSource.LoadMoviesCallback mLoadMoviesCallback;
```

Comment:
```suggestion
    private MoviesDataSource.LoadMoviesCallback mCallback;
```
nếu chỉ có 1 callback e nên viết ntn cho gọn